src/cursor.c

```diff
@@ -21,7 +21,7 @@ void cursor_move_up(Cursor* cursor) {
 }
 
 void cursor_move_down(Cursor* cursor) {
-    if (cursor->y == (LINES - 2)) { // Take box lines into account
+    if (cursor->y == (LINES - 5)) { // Take box lines into account
```

Comment:
These are to check against the border size and the status window size? Should those values be calculated and set in `frontend.c` or somewhere else?

Reply:
Since @MatthewBeaudouinLafon won't have time to assess this I'll merge it in now. I added the issue as #24.